src/ResourceManager/Compute/Commands.Compute/Microsoft.Azure.Commands.Compute.dll-Help.xml

```diff
@@ -13015,6 +13015,24 @@ PS C:\&gt; Set-AzureRmVMBootDiagnostics -VM $VM -Enable -ResourceGroupName "Reso
 <maml:uri /></dev:type>
 <dev:defaultValue>None</dev:defaultValue>
 </command:parameter>
+<command:parameter required="false" variableLength="true" globbing="false" pipelineInput="True (ByPropertyName)" position="named" aliases="none"><maml:name>Secret</maml:name>
+<maml:Description><maml:para>The encryption key used to encrypt and decrypt the data bag item values.
```

Comment:
Please make these changes to the markdoan file here:  https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Compute/Commands.Compute/help/Set-AzureRmVMChefExtension.md and add regenerate the help xml.